modules/virt-restarting-vm-web.adoc

```diff
@@ -23,14 +23,12 @@ To avoid errors, do not restart a virtual machine while it has a status of *Impo
 
 * To stay on this page, where you can perform actions on multiple virtual machines:
 
-.. Click the Options menu {kebab} located at the far right end of the row.
+.. Click the Options menu {kebab} located at the far right end of the row and click *Resume VirtualMachine*.
 
 * To view comprehensive information about the selected virtual machine before
 you restart it:
 
 .. Access the *VirtualMachine details* page by clicking the name of the virtual
 machine.
 
-.. Click *Actions* -> *Restart*.
-
-. In the confirmation window, click *Restart* to restart the virtual machine.
+.. Click *Actions* -> *Start*.
```

Comment:
Are these changes correct/intentional?

Reply:
I'm going to say that it is. Perhaps you could have QE verify?

Reply:
Some of it looks right (removing the confirmation window step) but I think there are some necessary changes. Please see https://github.com/openshift/openshift-docs/pull/69051#pullrequestreview-1940555314 

and yes, QE is needed here too